data/profileData.js

```diff
@@ -0,0 +1,17 @@
+const fakeUser =  {
```

Comment:
I appreciate you keeping this as similar as possible to the defined API response.